woof-code/rootfs-skeleton/usr/sbin/snapmergepuppy

```diff
@@ -215,14 +215,14 @@ do
 		#Check if the file was opened or shutting down
 		if [ "$(echo $OPF | grep -m1 "^$N\$")" == "" ] || [ "$SHUTDOWN" == "yes" ]; then
 		  cp -af "$N" "$BASE/$N" 2>>/tmp/snapmergepuppy-error
-		  rm "$N"
+		  [ $? -eq 0 ] && rm "$N"
```

Comment:
Copying will succeed even if a process is writing to the file (it will copy a potentially corrupt file), so this check doesn't do anything